### ISSUE TITLE
multiple inputs to the Dense Layer functional API

### ISSUE THREAD

**Issue by akmahaja** [fchollet/keras#2903]
How do we pass multiple inputs to the Dense Layer using the functional API and how is it interpreted in the Dense Layer code for the functional API ?

(Ref: Asking for something similar to what is done in the Graph Model, where one can pass two input layers to the Dense Layer)

**Comment by henry0312** [comment]
See http://keras.io/getting-started/functional-api-guide/#shared-layers

**Comment by akmahaja** [comment]
I am looking for something, where I can input two tensors and in the call function of the Dense Layer I could do the following.  In the above case it is more like concatenating the tensors and then passing them through the dense layer. How do I implement this scenario?

   def call(self, x, mask=None):
        output = K.dot(K.dot(x1, self.W),x2)

**Comment by henry0312** [comment]
I see.
Probably, you need to define your own layer (cf. http://keras.io/layers/writing-your-own-keras-layers/).
